### PR TITLE
fix(client/freeze): exit vehicle

### PR DESCRIPTION
**Pull request by Jaccosf**
- In my opinion, the best way to do this is let the player leave the vehicle (the vehicle will also stop, if the driver is being freezed) and freeze it after.

### CODE REVIEW

**Comment by Jaccosf** [comment]
This will be a fix for: #463

**Comment by NoobKeksTV** [comment]
yeah, this should work. 
#464 is not the right way, it would have to be run in a thread to be called every frame but this makes the ped leave any vehicle hes in so we can freeze him properly. 
Also we could just freeze the Player and the car both but as mentioned by tab in #463 that will require a check for the car when player gets unfreezed.

This gets approval by NoobKeks Pro dev 👍

**Comment by Jaccosf** [comment]
Haha thanks! 😃

File: scripts/menu/client/cl_freeze.lua [hunk 1]
```diff
@@ -27,6 +27,10 @@ end)
 
 RegisterNetEvent('txAdmin:menu:freezePlayer', function(isFrozen)
   debugPrint('Frozen: ' .. tostring(isFrozen))
-  FreezeEntityPosition(PlayerPedId(), isFrozen)
+  local playerPed = PlayerPedId()
+  if IsPedInAnyVehicle(playerPed) then
+    TaskLeaveAnyVehicle(playerPed)
```
Comment (TasoOneAsia) [inline scripts/menu/client/cl_freeze.lua]: Can we pass a flag of `16` to the optional flags please. This flag specifically specifies teleport outside of the vehicle immediately.

[Ref](https://docs.fivem.net/natives/?_0xD3DBCE61A490BE02)

**Review by TasoOneAsia — CHANGES_REQUESTED** [review]

File: scripts/menu/client/cl_freeze.lua (hunk 1 above)
Comment (Jaccosf) [reply scripts/menu/client/cl_freeze.lua]: Yeah sure

**Comment by TasoOneAsia** [comment]
I think the overall logic is this implementation is sound, I'll slate it for testing tomorrow, thanks for the patience.

**Pull request merged**